src/main/java/org/folio/services/domainevent/InstanceDomainEventPublisher.java

```diff
@@ -28,6 +28,7 @@
 
 public class InstanceDomainEventPublisher {
   private static final Logger log = getLogger(InstanceDomainEventPublisher.class);
+  private static final String JOB_ID_HEADER = "reindex-job-id";
```

Comment:
Consider renaming `JOB_ID_HEADER` to be more specific, for example `REINDEX_JOB_ID_HEADER`.

Reply:
Renamed to `REINDEX_JOB_ID_HEADER`